controllers/auth_config_controller.go

```diff
@@ -42,25 +42,25 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// ServiceReconciler reconciles a Service object
-type ServiceReconciler struct {
+// AuthConfigReconciler reconciles a AuthConfig object
+type AuthConfigReconciler struct {
 	client.Client
 	Log    logr.Logger
 	Scheme *runtime.Scheme
 	Cache  cache.Cache
 }
 
-// +kubebuilder:rbac:groups=config.authorino.3scale.net,resources=services,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=authorino.3scale.net,resources=authconfigs,verbs=get;list;watch;create;update;patch;delete
 
-func (r *ServiceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("service", req.NamespacedName)
+func (r *AuthConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	log := r.Log.WithValues("authConfig", req.NamespacedName)
 
-	service := configv1beta1.Service{}
-	err := r.Get(ctx, req.NamespacedName, &service)
+	authConfig := configv1beta1.AuthConfig{}
+	err := r.Get(ctx, req.NamespacedName, &authConfig)
 	if err != nil && errors.IsNotFound(err) {
 
 		// As we can't get the object, that means it was deleted.
-		// Delete all the services related to this k8s object.
+		// Delete all the auth configs related to this k8s object.
```

Comment:
I'm not sure about this, because if we are talking about "authorization configs" it's separated, but when talking about the CRD, it'd be `authconfigs`, no?. Not suggesting a change, just pointing it out, see if it makes more sense or not :)

Reply:
This one applies for any appearance of the term

Reply:
It does make sense, I guess. Thanks!